client/app/bundles/course/assessment/submission/components/answers/index.tsx

```diff
@@ -156,6 +171,12 @@ const SubmissionAnswer = <T extends keyof typeof QuestionType>(
 
       <Divider />
 
+      {Object.values(AttachmentErrorType).includes(error) && (
```

Comment:
`AttachmentErrorType` should be of a more generic error type so that it is extensible next time.

Reply:
changed into ErrorType